packages/ui/src/composites/wui-all-wallets-image/index.ts

```diff
@@ -1,31 +1,39 @@
 import { html, LitElement } from 'lit'
-import { customElement, property } from 'lit/decorators.js'
+import { customElement, property, state } from 'lit/decorators.js'
 import { globalStyles } from '../../utils/ThemeUtil'
 import '../wui-wallet-image'
 import styles from './styles'
-
-export interface WalletImage {
-  src: string
-  walletName: string
-}
+import type { IWalletImage } from '../../utils/TypesUtil'
 
 @customElement('wui-all-wallets-image')
 export class WuiAllWalletsImage extends LitElement {
   public static styles = [globalStyles, styles]
 
   // -- state & properties ------------------------------------------- //
-  @property({ type: Array }) public walletImages: WalletImage[] = []
+  @property({ type: Array }) public walletImages: IWalletImage[] = []
+
+  @state() public totalImages = 4
 
   // -- render ------------------------------------------------------- //
   public render() {
-    return this.walletImages.map(
-      ({ src, walletName }) =>
-        html`<wui-wallet-image
-          size="inherit"
-          src=${src}
-          walletName=${walletName}
-        ></wui-wallet-image>`
-    )
+    return html`${this.templateWalletImages()}`
+  }
+
+  private templateWalletImages() {
+    const renderedImages = this.walletImages
```

Comment:
We can simplify this i.e. loop through existing images and add them, then add placeholders for everything else, so

```
${renderedImages.map( w => html`...`)}
${[...Array(TOTAL_IMAGES - renderedImages.length)].map(w => html`...placeholder`)}
```